src/wowman/core.clj

```diff
@@ -919,24 +943,26 @@
 
 (defn-spec export-installed-addon ::sp/export-record
   "given an addon summary from a catalogue or .toc file data, derive an 'export-record' that can be used to import addon later"
-  [addon (s/or :catalog ::sp/addon-summary, :installed ::sp/toc), game-track (s/nilable ::sp/game-track)]
-  (let [stub (select-keys addon [:name :source :source-id])
+  [addon (s/or :catalog ::sp/addon-summary, :installed ::sp/toc)]
+  (let [game-track (:installed-game-track addon) ;; from toc file when addon installed/updated
+        stub (select-keys addon [:name :source :source-id])
         ;; when there is a catalog match, attach the game track as well
-        game-track (when (and (:source stub) game-track) {:game-track game-track})]
+        game-track (when (and (:source stub) game-track) ;; todo: this is too adhoc
```

Comment:
when *wouldn't* there be a :source attribute in either an :addon-summary or :addon ?